### PR TITLE
[QA-579]: Update Auth scripts based on latest changes.

### PR DESCRIPTION
**Pull request by abhinaysrivastavaperf**
### What?
Update Auth scripts based on latest changes

#### Changes:
- Updated signUp and signIn scenarios to use the new orch stub.

---

### Why?
Auth user journeys have a new stub as the entry point as part of latest changes. The script changes are to reflect these changes in the user journeys.

### CODE REVIEW

File: deploy/scripts/src/authentication/test.ts [hunk 1]
```diff
@@ -443,18 +417,69 @@ export function initializeJourney(groups: readonly string[]): Response {
   })
 }
 
+export function orchStubSubmit(groups: readonly string[]): Response {
```
Comment (Tom-Dann) [inline deploy/scripts/src/authentication/test.ts:420]: Is this function replacing the `initializeJourney()` function?

Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/authentication/test.ts:420]: Yes

Comment (Tom-Dann) [reply deploy/scripts/src/authentication/test.ts:420]: Okay I suggest removing that function as part of this PR in that case

Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/authentication/test.ts:380]: Done

File: deploy/scripts/src/authentication/test.ts [hunk 2]
```diff
@@ -567,25 +534,13 @@ export function signIn(): void {
...
       })
     })
   }
 
-  // 25% of users logout
-  if (Math.random() <= 0.25) {
-    sleep(1)
-
-    // B02_SignIn_07_Logout
-    logout(groups.slice(17, 21))
```
Comment (Tom-Dann) [inline deploy/scripts/src/authentication/test.ts:588]: Can remove the `logout()` function too if it is no longer being used

**Review by Tom-Dann — CHANGES_REQUESTED** [review]

File: deploy/scripts/src/authentication/test.ts (hunk 2 above)
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/authentication/test.ts:588]: Removed

**Pull request merged**